apps/happy-blocks/index.php

```diff
@@ -41,10 +41,22 @@
 function a8c_happyblocks_assets() {
 	$assets = require_once plugin_dir_path( __FILE__ ) . 'dist/editor.min.asset.php';
 
+	wp_register_script( 'a8c-happyblocks-pricing-plans', '', array(), '20221117', true );
+	wp_enqueue_script( 'a8c-happyblocks-pricing-plans' );
+	wp_add_inline_script(
+		'a8c-happyblocks-pricing-plans',
+		sprintf(
+			'window.A8C_HAPPY_BLOCKS_CONFIG = %s;
+			window.configData ||= {};',
```

Comment:
Could we rename the `configData` to have the happy blocks prefix? This looks quite generic, potential for a naming clash. Also, what does `||= {}` mean?

Reply:
I see.

Oh, nice, TIL.